src/utils/verificar-token.ts

```diff
@@ -0,0 +1,23 @@
+import * as jwt from 'jsonwebtoken';
+const secretKey = process.env.SECRET_KEY || 'seminario';
+import { Response, Request, NextFunction } from 'express';
+
+
+export const verificarToken = (req: Request, res: Response, next: NextFunction) => {
+  const token = req.headers['authorization'];
+
+  if (!token) {
+    return res.status(403).json({ message: 'Token no proporcionado' });
```

Comment:
please add the library https://www.npmjs.com/package/http-status-codes and use it, for the name you can separate the messages in a different file. check this example.
Organizing response messages in a separate file is a great way to manage and maintain your code, especially in a TypeScript project. Here's an approach you can use:

1. In a separated file you should create /src/busines/messages.ts 
2. Organize your messages in a way that makes sense for your application. You could categorize them based on features, error types, etc
3. Import the messages into the files where you need them and use them as required.
```
// messages.ts

const authMessages = {
    tokenMissing: 'Unauthorized: No token provided',
    tokenInvalid: 'Unauthorized: Invalid token',
    // ... other authentication related messages
};

const userMessages = {
    userNotFound: 'User not found',
    userCreated: 'User successfully created',
    // ... other user management related messages
};

export { authMessages, userMessages };
```
How you can use.

```
response.status(HttpStateCodes.UNAUTHORIZED).json({authMessages.tokenMissing});
```